src/utils/file.js

```diff
@@ -76,3 +80,64 @@ export function createObjectPathFromFilePath(filePath, root) {
     const parentDirs = root ? path.relative(root, dirname).split(path.sep) : [];
     return [...parentDirs, basename].map((item) => `['${item}']`).join("");
 }
+
+/**
+ * Reads asset file and stores asset data in target object under object path which reflects the file system.
+ * @param {Object} targetObject - Object in which asset data should be stored
+ * @param {string} assetPath - Absolute path to asset file.
+ * @param {string} assetRoot - Path to asset root directory to construct relative path.
+ */
+export function loadAndInsertAssetData(targetObject, assetPath, assetRoot) {
```

Comment:
Should this file be in TypeScript already? The rest of the repo already is, it seems

Reply:
It seems there are still a couple of JS files in `src/utils`: https://github.com/Exabyte-io/code.js/tree/main/src/utils